scopes/code/ui/code-tab-tree/code-tab-tree.module.scss

```diff
@@ -9,10 +9,10 @@
 .codeTabDrawer {
   > div:first-child {
     border: none;
-    border-bottom: 1px solid var(--border-color);
+    border-bottom: 1px solid var(--bit-border-color-lightest);
```

Comment:
can we add fallbacks to all the colors which dont have it? looks like the time to do it

Reply:
yes, but I'm not going to go over all the code and add it retroactively

Reply:
```suggestion
    border-bottom: 1px solid var(--bit-border-color-lightest, #ededed);
```